automation/tables.py

```diff
@@ -11,3 +11,8 @@ class Table(enum.Enum):
         model_cls=models.InboundModel,
         status_to_cb={None: inbound.on_new},
     )
+    VOLUNTEERS = airtable.TableSpec(
```

Comment:
Sounds good. I'll try and fold the table specs into the model classes themselves.

Reply:
Mmm one issue is that the status to callback mapping probs shouldn't be defined in `models.py` because that create a circular dependency between `models.py` and the `functions/...` modules.